syntax = "proto3";

package Packet;

message NewTurnPacket {
	uint32 current_player_id = 1;
	uint32 current_turn = 2;
}